src/pages/home/report/AnimatedEmptyStateBackground.js

```diff
@@ -32,13 +37,14 @@ function AnimatedEmptyStateBackground() {
         return {
             transform: [{translateX: withSpring(-IMAGE_OFFSET_X - xOffset.value)}, {translateY: withSpring(yOffset.value)}],
         };
-    });
+    }, []);
```

Comment:
This prevents unnecessary re-render of background image. Avoiding dependency array causes the style to be recomputed every time component changes. It was causing visual artefacts with resize mode.

Reply:
@aswin-s  Can you attach video of what issue you are facing if this is not present?

Because according to docs it is not required - https://docs.swmansion.com/react-native-reanimated/docs/core/useAnimatedStyle/#arguments

Reply:
Here you go. Without an empty dependency array the resize mode is not reflected properly.

https://github.com/Expensify/App/assets/6880914/8ebf8aac-7e56-4c6e-9aa2-70d55e013d03

Reply:
Got it. It seems without adding empty dependencies, it is causing issues.